generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -190,7 +201,7 @@ import java.util.stream.Stream;
   <%_ } _%>
 <%_ } _%>
 import java.util.List;
-<%_ if (fieldsContainUUID || primaryKey.typeString || otherEntityPrimaryKeyTypesIncludesUUID) { _%>
+<%_ if (fieldsContainUUID || primaryKey.typeString || primaryKey.fields.map(f => f.fieldType).some(t => ['String', 'UUID'].includes(t)) || otherEntityPrimaryKeyTypesIncludesUUID) { _%>
```

Comment:
Update fieldsContainsUUID and add derived primaryKeyContainsString (not sure primaryKeyTypeString is useful anymore)